content/docs/concepts/metric_types.md

```diff
@@ -87,7 +87,7 @@ during a scrape:
 
   * streaming **φ-quantiles** (0 ≤ φ ≤ 1) of observed events, exposed as `<basename>{quantile="<φ>"}`
   * the **total sum** of all observed values, exposed as `<basename>_sum`
-  * the **count** of events that have been observed, exposed as `<basename>_count`
+  * the **count** of events that have been obserlabelved, exposed as `<basename>_count`
```

Comment:
I don't think you changed this on purpose.

Reply:
@avivace would you fancy to update your PR with a fix so that we can merge this?